### DISCUSSION TITLE
심화프로젝트 4주차 팀회고

### DISCUSSION THREAD

**Discussion by yeji0512** [codeit-bootcamp-frontend/2-Weekly-Mission#445] [회고]
**벌써 4주차가 마무리 되어가네요:)
여러분들이 매주 올려주시는 회고 내용을 보며, 정말 고생 많으시다고 말씀드리고 싶었어요 💜
남은 프로젝트 기간도 꼭 건강 잘 챙겨가며, 즐겁게 달려보아요 🏃‍♀️**

# 왜 회고를 해야하나요?
- **원활한 팀 프로젝트 진행을 위해, 회고가 좋은 도구가 되어줄거예요.** 
매주 팀 단위로 회고를 하며 어떤 점이 좋았고, 어떤 점이 아쉬웠고, 아쉬웠던 부분은 어떻게 개선해볼지 함께 이야기해보면 
그 다음주에는 더 나은 프로젝트를 진행할 수 있을거예요. 회고를 통해 함께 성장해보아요 🔥 
- **회고 시간만큼은 솔직한 생각, 마음을 터놓아보기! 😎** 
솔직한 회고를 통해 다음주는 더 즐겁게 프로젝트를 진행할 수 있을거예요.  

# 회고 방법
![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/144666341/b1f1cc83-7d24-4156-910d-3fc9b6bb7555)
회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다. 
- 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
    - Keep: 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
    - Problem: 현재 우리팀이 가지고 있는 문제
    - Try: Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
- [KPT 회고 피그마 템플릿](https://www.figma.com/file/0NHIhlM9vhYZDrtZa7nJn8/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF?type=whiteboard&node-id=0%3A1&t=p6vXDVEjQntHKaNP-1)을 활용해서, 팀 회고를 진행하셔도 됩니다. 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요. 
- **이번주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡처본을 올려주셔도 돼요.)

# 참고 글
- [KPT 회고란 무엇인가?](https://brunch.co.kr/@jinha0802/35)
- [팀문화의 탄생](https://techblog.woowahan.com/2677/)

**Comment by muyahho** [comment]
### 4팀
### Keep
먼저 작업이 끝난 팀원들이 적극적으로 추가 작업을 하거나 다른 팀원을 도와주며 협력을 잘 해나가고 있다.
매일 회의록을 작성합 입니다.
소통이 원할히 진행됩니다.
문제 해결을 위한 협력이 활발합니다.
변함없이 열심히 하려고 합니다.
### Problem
컨벤션들을 제대로 지키지 않은 부분이 있는것 같습니다.

### Try
컨벤션 제대로 지키기
적극적으로 코드 리뷰 하기

마지막까지 화이팅!!

**Comment by JISOuO** [comment]
# 10팀😎
## Keep
- 날마다 회의록을 작성해서 어떤걸 알고있어야하는지 파악이 빠르다
- 본인이 맡은 부분이 아니더라도 다함께 해결하고있다
- 수정사항을 적극적으로 제시함
- 문제가 생길 때 트러블 슈팅으로 작성해 문제 파악과 해결법을 공유한다

## Problem
- 협업 프로젝트가 처음이라 진행 속도에 대한 불안함이 있음

## Try
- PR 코드 리뷰 시 댓글을 달면 더 좋을 것 같음
- zustand와 react query를 빨리 사용하고 싶음

마지막 파트는 10팀이 점령하겠습니다+_+

**Comment by mijin0928** [comment]
## 5팀
### Keep
- 처음 프로젝트 시작했을 때보다 소통이 빨라졌다
- 매일 회의록을 작성한다.
- 트러블 슈팅을 작성한다.

### Problem
- pr 확인이 늦다

### Try
- pr 올리고 22시간 안에 확인하기 
- 프로젝트 마무리까지 열심히 하기!

![doggo-puppy](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/94034865/12e3762f-9c08-4bb0-a4c7-5af00ecc8dbf)

**Comment by CheeseB** [comment]
# 3팀 🦸🏼 

![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/43297823/ec77b7b8-4326-4068-a62d-31d009d5c4bc)

**Comment by 9ummy9** [comment]
# 6팀

![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/41e8405c-2753-4f01-a624-6bcdfb44de86)


----

![tumblr_maj48b1RWL1rd8fnlo1_500](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/0f139f97-e68e-4da8-a7a7-c2012784aa93)

항상 건강이 첫번째입니다...
남은 2주간 아프지말고 건강하게 프로젝트 잘 마쳤으면 좋겠습니다! 6팀 모두 파이팅!

**Comment by MoonPillGyeom** [comment]
## 8팀
## KEEP
- 소통이 잘 됩니다
- 데일리 미팅이 잘 진행되고 있다
- 문제가 있을때 서로 도움을 잘 주고 받는다
- 각자 맡은 부분에 책임을 가지고 작업을 한다

## PROBLEM
- 지난 한 주간 작업 속도가 조금 느렸다
- 단위 별 목표가 안 잡혀 있어 진행 속도가 체감이 잘 안된다
- 진행 속도가 느리다

## TRY
- 다들 화이팅!
- 활발한 소통과 협력
- 구현이 급하니 눈 앞에 있는것 부터 차근차근 열심히 하다보면 괜찮아 질것같다
- 단위별 목표를 설정하기

**Comment by SeanKim-416** [comment]
# 11팀

![스크린샷 2024-03-25 오전 10 57 42](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/56223156/abde6e5a-ff79-44dc-a61f-83692476b423)
<hr>

![스크린샷 2024-03-25 오전 10 57 50](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/56223156/26ceb71b-017d-4695-94a2-a8574af4bd25)
<hr>

![스크린샷 2024-03-25 오전 10 58 09](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/56223156/ebdd33c5-254f-4eb7-a771-244bf6f2111e)

**Comment by 00TaciTa00** [comment]
# Keep
- 매일 팀미팅을 해서 서로의 진행 상황 공유가 잘 되고 있습니다.
- 서로 모르는 부분을 같이 해결해주는 점이 좋아요.

# Problem
- 모르는 거 있을 때 너무 시간 끌지 말기!

# Try
- 기획 때 결정된 부분들이 기억 안 나거나 의문이 생기면 먼저 노션을 확인하고, 팀미팅 때 먼저 물어보셨으면 합니다.
- 의견 제시는 바로바로 해주세요.

**Comment by rladngus133** [comment]
## 2팀 👊
### Keep
- 팀 미팅시 말없는 불참이 없고 다들 잘 참여해준다.
- 각자 궁금한점에 대하여 빠른 공유 및 피드백이 잘 이루어진다.
- 기능단위의 PR이 이루어져 각 PR 마다 비교적 세세한 설명을 적어주어 이해하기가 쉽다.
- 새로운 기술을 계속하여 도입하려고 해본다.

### Problem
- 새로 접하는 기술이 많다보니 살짝 더뎌지는 감이있다.
- 위의 이유와 더불어 각자 맡은 페이지/기능내에서 많은 시간이 필요하다보니 코드리뷰가 줄어가는 감이있다.

### Try
- Keep에서 잘 진행되고 있는 내용과 같이 Q&A를 적극적으로 해보자

**Comment by juncastle97** [comment]
# 9팀

## keep
- 매일 소통을 잘 하고 있다.
- 각자 맡은 부분을 열심히 해내고 있다.
- 기능에 대해 이야기 할 때 서로의 의견을 존중해준다.

## problem
- 매일 한 두명씩 지각을 한다.

## try
- 팀미팅 시간, 전체 회의 시간을 잘 지키자.
